validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, database, state_delta_store):
 
         self._address_regex = re.compile('^[0-9a-f]{70}$')
 
+        self._namespace_regex = re.compile('^[0-9a-f]{0,70}$')
```

Comment:
Just to make sure, @jsmitchell , can you provide the reasoning on why namespaces must be even number of characters?